client/lib/utils.js

```diff
@@ -22,6 +22,11 @@ Utils = {
     return string.charAt(0).toUpperCase() + string.slice(1);
   },
 
+  isMobile() {
+    // return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
+    return $(window).width() <= 800;
```

Comment:
I think it is better to rely on the device size than the user agent indeed. So you should remove the comment above.

Also we should use Meteor reactivity here! The window width may change (eg if a browser user change resize the window, or with the multi-app mode of new iOs devices). The way to support that is to register a tracker dependency like the one I created [here](https://github.com/wekan/wekan/blob/677e9937e0c1872bfe4ec5ce87bf2a8ca99febe1/client/lib/popup.js#L5-L6) and used [here](https://github.com/wekan/wekan/blob/677e9937e0c1872bfe4ec5ce87bf2a8ca99febe1/client/lib/popup.js#L163).

Actually since the dependency is actually the same as in the Popup example, you could move its declaration in the Utils and use it like `Utils.windowWidthDep.depend();`.

Feel free to ask any question if something isn’t clear :)